Libplanet.Tests/Action/ActionContextTest.cs

```diff
@@ -18,6 +18,7 @@ public void RandomShouldBeDeterministic()
                 var context = new ActionContext(
                     from: new Address("21744f4f08db23e044178dafb8273aeb5ebe6644"),
                     to: new Address("21744f4f08db23e044178dafb8273aeb5ebe6644"),
+                    blockIndex: 1,
```

Comment:
It would be better that adding blockIndex related assertion in here (or, adding other test case seems also good.)

Reply:
@longfin Done and done